extensions/desktop/common/init

```diff
@@ -71,7 +71,9 @@ export SNAP_LAUNCHER_ARCH_TRIPLET="$ARCH"
 
 # Don't LD_PRELOAD bindtextdomain for classic snaps
 if ! grep -qs "^\s*confinement:\s*classic\s*" "$SNAP/meta/snap.yaml"; then
-  if [ -f "$SNAP/lib/$ARCH/bindtextdomain.so" ]; then
+  if [ -f "$SNAP/gnome-platform/lib/$ARCH/bindtextdomain.so" ]; then
+    export LD_PRELOAD="$LD_PRELOAD:$SNAP/gnome-platform/\$LIB/bindtextdomain.so"
```

Comment:
why are you escaping $LIB?

Reply:
> why are you escaping $LIB?

That was copied from what was already there which is now in the elif.  Maybe we don't need that.